README.md

```diff
@@ -105,6 +105,37 @@ To regenerate the swagger client:
 3. Run `java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -i http://localhost:8080/swagger.json -l java -o <output directory> --library jersey2`. The output directory is where you have dockstore/swagger-java-client/.
 4. NOTE: Rengenerating the swagger client will probably generate an incorrect pom file. Use git checkout on the pom file to undo the changes to it.
 
+## Swagger Java Client for quay.io
+
+Background:
+
+ * Client library generated by the [swagger code generator](https://github.com/swagger-api/swagger-codegen)
+ * Is generated based on the quay.io's swagger UI spec
+ * Used by the Dockstore CLI to make http requests to quay.io
+ * If CoreOS changes their API, you will need to regenerate the swagger client.
+ 
+ To regenerate the swagger client:
+ 
+1. Run `echo "{\"library\":\"jersey2\",\"invokerPackage\":\"io.swagger.quay.client\",\"modelPackage\":\"io.swagger.quay.client.model\",\"apiPackage\":\"io.swagger.quay.client.api\"}" > config.json`
+2. Run `java -jar modules/swagger-codegen-cli/target/swagger-codegen-cli.jar generate -i https://quay.io/api/v1/discovery -l java -o <output directory> --library jersey2`. The output directory is where you have dockstore/swagger-java-client/.
```

Comment:
Kind of. These commands are run in the swagger-codegen directory, so it kinda depends where the two directories are in relation to each other, so I left it as `<output directory>` for the developer to fill in.